### PR TITLE
created git module for c-lightning-api plugin

### PR DESCRIPTION
**Pull request by nettijoe96**
submodule for c-lightning-api

### CODE REVIEW

**Comment by cdecker** [comment]
Hi @nettijoe96 thanks for adding the project as a submodule. The name is a bit confusing imho since it is first and foremost a graphql adaptor, so maybe we should call it `graphql` or something to that end?

**Comment by nettijoe96** [comment]
just updated the name

**Comment by cdecker** [comment]
Why the `c-lightning` prefix? These are all `c-lightning` plugins :wink:

**Comment by nettijoe96** [comment]
in case someone comes across the repo outside of this plugin repo

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,6 @@
 [submodule "sitzprobe"]
 	path = sitzprobe
 	url = https://github.com/niftynei/sitzprobe
+[submodule "c-lightning-graphql"]
```
Comment (cdecker) [inline .gitmodules]:
```suggestion
[submodule "graphql"]
```

File: .gitmodules [hunk 2]
```diff
@@ -1,3 +1,6 @@
...
+	path = c-lightning-graphql
```
Comment (cdecker) [inline .gitmodules]:
```suggestion
	path = graphql
```

**Comment by nettijoe96** [comment]
wow that is cool! didn't know you could do that!

**Comment by cdecker** [comment]
ACK https://github.com/lightningd/plugins/pull/7/commits/3ff173b405355995cf5c31b83b003b5fdafa29a9

**Pull request merged**